M/MKL/build_tarballs.jl

```diff
@@ -13,8 +13,8 @@ platform_sources = [
     (
         platform = Platform("x86_64", "linux"; libc="glibc"),
         source = ArchiveSource(
-            "https://anaconda.org/intel/mkl/2021.1.1/download/linux-64/mkl-2021.1.1-intel_52.tar.bz2",
-            "bfb0fd056576cad99ae1d9c69ada2745420da9f9cf052551d5b91f797538bda2";
+            "https://anaconda.org/intel/mkl/2022.0.1/download/linux-64/mkl-2022.0.1-intel_117.tar.bz2",
```

Comment:
This is 2022.0.1 instead of 2022.0.0?

Reply:
Release notes shows that the lastest version is `2022.0.0`.
But that for `linux32/64` has been marked as `deprecated`.
So I picked the lastest one.